src/main/java/org/nuxeo/segment/io/SegmentIODataWrapper.java

```diff
@@ -111,8 +103,8 @@ protected boolean isAllowed(Object value) {
             if (key.startsWith(GROUP_KEY_PREFIX)) {
                 String gKey = key.substring(GROUP_KEY_PREFIX.length());
                 Serializable value = metadata.get(key);
-                if (value!=null) {
-                    if ( isAllowed(value)) {
+                if (value != null) {
+                    if (isAllowed(value)) {
```

Comment:
The code in lines L106-L114 are duplicate with L86-94, please consider if you can refactor them.